### PR TITLE
Documentation: fix formatting

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Documentation: fix formatting

Testing my script, found several places to be improved.
Also noticed that `ecs_*` modules documentation should be rewritten significantly.

I also suggest using
```
ansible.builtin.debug:
  var: account_data.account.contact
```
instead of
```
ansible.builtin.debug: var=account_data.account.contact
```
as recommended by the documentation

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
modules

### CODE REVIEW

**Comment by MarkusTeufelberger** [comment]
The `var=foo` vs. `var: foo` syntax would be a great addition for `ansible-lint` btw.

File: plugins/modules/acme_account_info.py [hunk 1]
```diff
@@ -55,9 +56,9 @@
     that:
       - account_data.exists
 - name: Print account URI
-  debug: var=account_data.account_uri
+  ansible.builtin.debug: var=account_data.account_uri
```
Comment (felixfontein) [inline plugins/modules/acme_account_info.py:59]:
```suggestion
  ansible.builtin.debug:
    var: account_data.account_uri
```

File: plugins/modules/acme_account_info.py [hunk 2]
```diff
@@ -55,9 +56,9 @@
...
 - name: Print account contacts
-  debug: var=account_data.account.contact
+  ansible.builtin.debug: var=account_data.account.contact
```
Comment (felixfontein) [inline plugins/modules/acme_account_info.py:61]:
```suggestion
  ansible.builtin.debug:
    var: account_data.account.contact
```

File: plugins/modules/acme_account_info.py [hunk 3]
```diff
@@ -69,7 +70,7 @@
     that:
       - account_data.exists
 - name: Print account contacts
-  debug: var=account_data.account.contact
+  ansible.builtin.debug: var=account_data.account.contact
```
Comment (felixfontein) [inline plugins/modules/acme_account_info.py:73]:
```suggestion
  ansible.builtin.debug:
    var: account_data.account.contact
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@Andersson007 thanks for fixing this!
@MarkusTeufelberger thanks for reviewing!

**Pull request merged**